component/main.jsonnet

```diff
@@ -38,9 +38,30 @@ local db_secret = kube.Secret(params.database.secretname) {
   stringData: secrets[params.database.provider],
 };
 
+local keycloak_tls = {
+  apiVersion: 'cert-manager.io/v1alpha2',
+  kind: 'Certificate',
+  metadata: {
```

Comment:
I'm not sure why this is needed. Why not just add the cert-manager annotations to the ingress?

Reply:
Because it is confusing having an ingress object in managing the certificate, but is not using it. Certificate reloads require a Keycloak restart and are not done with a ingress object reload. I'm also not sure, if the certificate renew / certificate validation does work, when the ingress traffic is passed directly to Keycloak.

Reply:
The certificate object is now created depending on `tls.variant`is `certmanager` and `tls.termination`is `passthrough`.